src/net/sourceforge/kolmafia/KoLmafia.java

```diff
@@ -267,11 +270,7 @@ public static final void main(final String[] args) {
       }
     }
 
-    // All dates are presented as if the day began at rollover.
-
-    TimeZone koltime = TimeZone.getTimeZone("GMT-0330");
-
-    KoLConstants.DAILY_FORMAT.setTimeZone(koltime);
+    KoLConstants.DAILY_FORMAT.setTimeZone(kolTimeZone);
```

Comment:
The variable koltime was only used to set the format TimeZone and so it is no longer needed.